recipes/fastlinkingnumbers/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "fastlinkingnumbers" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/antequ/fastlinkingnumbers/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: b3393bffb8669ab8214b1b86ae8b1d48abd70ba021134a3f2a5e5fcfd641e7d9
+
+build:
+  number: 0
+  skip: True  # [win or osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```

Comment:
I don't see any C sources. Are you sure you need the C compiler?

Reply:
Removed!